src/vl-mapactions-delete-action.js

```diff
@@ -30,20 +30,24 @@ export class VlDeleteAction extends VlBoxSelectAction {
         layer.getSource().removeFeature(feature);
       }
     };
+    
+    const afterOnDelete = () => {
+    	this.clearFeatures();
+    	this.map.render();
+    };
 
     super(layer, (features) => {
       if (onDelete && onDelete != null) {
         onDelete(features, (feature) => {
           removeFeature(feature);
-          this.clearFeatures();
         }, () => {
-          this.clearFeatures();
         });
+        afterOnDelete();
```

Comment:
Moet deze niet in de onDelete functie?

Reply:
nee denk het niet, hij gaat voor elke "geselecteerde" feature nog ne callback doen waarin de client kan beslissen of die feature effectief mag removed worden ja of nee, en dan daarna kan em re-renderen ...

Reply:
het voelt nu wat aan dat dit een race condition kan geven als afterOnDelete() voor removeFeature() uitgevoerd wordt. Maar dat leid ik enkel af uit hoe ik het hier nu zie staan, dus kan zeker mis zijn.

Reply:
Het 'daarna' zou pas mogen als de afnemer de onDelete success/decline functie heeft opgeroepen, dus in de callback. Eventueel kan de naamgeving ook nog wat beter dan omdat in het geval zonder onDelete ook nog afterONDELETE wordt gecalled. Eventueel clearFeaturesOnMap of zo?